nb2/service/dtos.py

```diff
@@ -42,9 +63,22 @@ class AddQuoteDTO(BaseDTO):
     Data transfer object for adding a Quote to a Person.
 
     Required Fields:
-        slack_user_id: String representing the unique Slack identifier for a Person.
         content: string representing a Quote the Person has said.
     """
 
-    slack_user_id: str
+    person: Person
+    content: str
+
+
+@dataclass
+class AddGhostQuoteDTO(BaseDTO):
```

Comment:
@karally do you think we need this DTO? It looks like adding a quote is done without using this because even if the ghost person isn't a slack user they are still returned as a Person

Reply:
yea doesn't look like this is used anywhere and ghosts are people too :ghost: I think we can remove this